### DISCUSSION TITLE
Congrats! Your request for @yourname was approved and your benefits will be available soon

### DISCUSSION THREAD

**Discussion by Crysil25** [community/community#58094] [GitHub Education] [answered]
### Hi everyone,

Hello good day ma'am/sir. Sorry about my grammar im not good in English. Please bare be with me
I have two questions
1. How many days when my benefit Will be available soon?? I was approved last June 11, 2023 or lets say 4 days from now my benefits is not yet available. Please give me an idea when the maximum date my benefits will be released?? 
![IMG_20230615_093547_257](https://github.com/community/community/assets/135818863/a3614011-6cae-49d2-a943-82209fca5cd6)

And my second question
2. It is ok ? my friend asked me to use my celpon when they applying to github for students?? I have a classmate on my school asked me or asked me a favor to borrow my phone. Let's say my phone camera is good for capturing their ID when the github asking their Academic ID. They have Thier own phone but they always rejected due to blurd capturing. So they asked me to borrow my phone. Is it not bad they register Thier identity to my phone?? I was afraid if github banned my account since my classmates use my phone to verify thier account. But of course we have different account or Gmail, or name. I was afraid only of github system detects that my phone registered many account. 

Please answer all my questions 
That's all and Thank you.

**Comment by Crysil25** [comment]
Please help

**Reply by jge162** [reply]
I believe the verification will be based on your username and account not that two people used the same phone. Also you should be getting access probably any min now. It took me about 3 days to get access and I never got an email.

**Reply by Crysil25** [reply]
Ok thanks man. I hope today i will access my benefits

**Reply by Oswaldshoo** [reply]
did you get approved?

**Reply by Dhruv35M** [reply]
what should I do, I didn't get approved, it's my 8th attempt.

**Comment by StefanDespotovic** [comment]
> Ok thanks man. I hope today i will access my benefits
if it is ok with u can you comment when they give you benefits? I have applied two days ago and still waiting

**Reply by Crysil25** [reply]
yes brother. but until now still not yet

**Reply by Crysil25** [reply]
I already get my benefits, takes 5 days

**Reply by jge162** [reply]
Wow must be more people applying now hence the longer wait. Congrats

**Reply by Crysil25** [reply]
Thanks, maximum duration they give the benefits is 5 days

**Reply by Kyawgyi99999** [reply]
Love you too Google docs create and attach it wave pay account

**Comment by littlebear1234** [comment]
I have the same problem with you. My approval date is June 12th, but so far I still can't use mine

**Reply by Crysil25** [reply]
I already get my benefits last day, takes 5 days

**Reply by Kyawgyi99999** [reply]
Sent to me account email

**Reply by maruf7705** [reply]
YOUR ACCOUNT ZERO ACTIVITY

**Comment by StefanDespotovic** [comment]
I have got it now

**Reply by Kyawgyi99999** [reply]
Sent from beta version as love to get money me Access to verify account

**Comment by abukhalidsajjad** [comment]
I got the benefits within 5 days, after account approval.

**Reply by Nullptr2109** [reply]
Will I receive an email after receiving benefits?

**Comment by shanravins0** [comment]
Congrats! Your request for [@shanravins0](https://github.com/shanravins0) was approved and your benefits will be available soon.

This is the message showing in https://education.github.com/discount_requests/application

But I haven't received notthing. My application has been on March 1 and till now didn't received anything. Is there something else i need to do. ??  please help

**Reply by JavaScriptonit** [reply]
gz

**Comment by budennovsk** [comment]
![image](https://github.com/community/community/assets/97764479/e8122340-bf68-465f-b75b-c286ba5f8ca2)
Hi guys! I sent my data for verification and everything went without errors! It's been 5 days so far without success! How will they get through ? do you have any feedback???

**Reply by Raiden129** [reply]
Its been 3 days for me sigh

**Reply by JavaScriptonit** [reply]
sigh

**Reply by budennovsk** [reply]
I was added! All CHEERS . 9 days were given access! patience to all

**Reply by pavankandati** [reply]
oops 9 days that really a long, i am still waiting for mine

**Reply by Akaki2003** [reply]
im waiting for 2 months

**Reply by pavankandati** [reply]
mine is completed within 7 days

**Reply by MarvelousZK** [reply]
I hope today i will access my benefits, Im waiting for 4days……

**Comment by JavaScriptonit** [comment]
I hope today i will access my benefits

**Reply by budennovsk** [reply]
маякни! если получиться получить доступ тут напиши!

**Reply by deGekata** [reply]
![image](https://github.com/community/community/assets/76394585/40686616-0872-4718-ad9f-7724166c4f49)
Кайфуем. Надеюсь через пару дней хотя бы придёт. Подавал 18.10.2023

**Reply by parkerpigeon** [reply]
I hope today i will access my benefits too

**Comment by maruf7705** [comment]
![image](https://github.com/community/community/assets/113976731/dca8069d-e414-47ee-95a4-d6adbc6ed11c)
I was submitted 2 September

**Reply by elhajoujy** [reply]
i have the same issue as you I submitted my proposal yesterday ! 7Sep

**Reply by elhajoujy** [reply]
I checked your profile, and your benefits are available due to the pro status on your profile!

**Reply by maruf7705** [reply]
yes, they give  . What about you ?

**Comment by AlongWY** [comment]
i have the same issue and i submitted my proposal in 12 Sep.

**Reply by maruf7705** [reply]
you have pro

**Reply by Oswaldshoo** [reply]
I have the same issue.

**Reply by grek12** [reply]
I have the same problem, on hold since September 12.

**Reply by rasakahmed** [reply]
> i have the same issue and i submitted my proposal in 12 Sep.

Same. I have applied on Sept 13. Haven’t got yet.

**Reply by HanYncBWH** [reply]
I got a same problem too, I was approve on Sept 13 and have't got my benefit.

**Reply by Oswaldshoo** [reply]
I approved today.. looks like it takes 3 to 4 day

**Reply by kowama** [reply]
Same delay for me,  about 3 days.

**Reply by pavankandati** [reply]
I applied for it just today and got approved. I need to see how long they will take to provide benefits

**Reply by AhmedAbouelkher** [reply]
@pavankandati 
I got approved today, when did you get your benefits?

**Reply by vins-anity** [reply]
@Oswaldshoo 
did you pass a Dated School ID?

**Reply by Oswaldshoo** [reply]
I'm still in university

**Comment by SilenceMonk** [comment]
I waited for 10 days straight and got approved

**Reply by vins-anity** [reply]
did you pass a dated school ID?

**Comment by ghost** [comment]
i have the same issue and i submitted my proposal in 19 Oct.

**Reply by deGekata** [reply]
Im waiting with you)
Got approved just immediatly (i have been using students pack for already two years and now it asked for relevant information of education status)
Also i dont know if i need to translate my document to english...

**Comment by hao0190** [comment]
Fix this Error <Congrats! Your request for ***** was approved and your benefits will be available soon.

**Comment by hao0190** [comment]
Fix this Error! Your request for @hao0190 was approved and your benefits will be available soon.

**Reply by pavankandati** [reply]
wait for 7 days

**Comment by veje95** [comment]
Approved Submitted Oct 24, 2023
Congrats! Your request for [@veje95](https://github.com/veje95) was approved and your benefits will be available soon.

I've been waiting for 2 days too, I hope it won't take much longer.

**Reply by eos-evm** [reply]
Have you been approved?

**Reply by pavankandati** [reply]
wait for 7 days

**Comment by ShejanMahamud** [comment]
Approved Submitted Nov 3, 2023
Congrats! Your request for @ShejanMahamud was approved and your benefits will be available soon.

When it will be? Now it's 3 days done

**Reply by pavankandati** [reply]
7 days

**Comment by eos-evm** [comment]
I'm not very good at English

"I encountered a similar issue to yours, but mine is even stranger. When I was applying, I was directly taken to the final step where I had to upload proof documents. I didn't see the page for selecting a school, choosing an email, or providing reasons for my application. I registered my account for the first time on October 28, 2023. However, in my application records, I have a failed student package application from the 27th. After the initial application failure, I uploaded proof documents for the second time on November 5th. Since then, I haven't received any emails, but on the application page, it says that I'm about to receive benefits. However, it shows that I was approved on November 1st, which doesn't match the timeline of my application. Up to this point, I still haven't received the benefits."

**Reply by MeMeTaTa** [reply]
So am I, and I noticed a strange thing that I haven't the page of providing reasons too. But the error message included "Please select proof type 'Other' for this image". I'm really confused by the 'proof type'.

**Reply by MeMeTaTa** [reply]
And github.com tell me my application has been approved, my benefits will be available soon. However, I still haven't received nothing. I was approved on yesterday, it's about 30 hours ago.

**Reply by pavankandati** [reply]
wait for 7 days

**Comment by Respawn-ID** [comment]
Same issue and still waiting....for 3days

**Reply by khoasv89** [reply]
i have been waiting for 5 days :D , i don't know how long it take

**Reply by hellojeremyonly** [reply]
@khoasv89 how did it go?

**Comment by ShejanMahamud** [comment]
Hello Everyone , i got approved after 3 days

**Comment by SpeedyAstro** [comment]
i am waiting for me also

**Comment by hellojeremyonly** [comment]
![Approve](https://github.com/community/community/assets/128933625/3177c45b-4672-478f-92b5-f2b242cd7b12)

Finally approved, and waiting..!

**Comment by Respawn-ID** [comment]
l have been approved , you need wait for 5days at least.....



---Original---
From: "Anubhav ***@***.***&gt;
Date: Thu, Nov 30, 2023 23:39 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Congrats! Your request for @yourname wasapproved and your benefits will be available soon (Discussion #58094)




 
i am waiting for me also

**Comment by Respawn-ID** [comment]
yes , GitHub will send you an email to let y
ou know that you have received benefits.



---Original---
From: ***@***.***&gt;
Date: Sun, Dec 3, 2023 17:53 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Congrats! Your request for @yourname wasapproved and your benefits will be available soon (Discussion #58094)




 
Will I receive an email after receiving benefits?

**Comment by hellojeremyonly** [comment]
I just got approved. Waited 4 days. 👍

**Comment by Respawn-ID** [comment]
Congrats!



---Original---
From: "Jeremy Chong Jia ***@***.***&gt;
Date: Mon, Dec 4, 2023 19:47 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Congrats! Your request for @yourname wasapproved and your benefits will be available soon (Discussion #58094)




 
I am just approved. Waited 4 days. 👍

**Comment by SJPBP** [comment]
I just got approved on Dec 30 and I hope to receive the benefit on January 1st as a welcome to the new year 🎉

**Reply by SJPBP** [reply]
And I wish you all a Happy New Year 🎆

**Reply by CberYellowstone** [reply]
have you received the benefit?

**Comment by Respawn-ID** [comment]
lol,thx




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "community/community"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月1日(星期一) 下午3:45
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [community/community] Congrats! Your request for @yourname was approved and your benefits will be available soon (Discussion #58094)





 
And I wish you all a Happy New Year 🎆

**Comment by Dhruv35M** [comment]
what should I do, I didn't get approved, it's my 8th attempt.

**Reply by SpeedyAstro** [reply]
You should try more

**Reply by JavenQvQ** [reply]
I tried 29 times before passing

**Comment by liuliangsir** [comment]
![education github com_discount_requests_application](https://github.com/community/community/assets/8049878/f8c62df7-a7ee-409c-8568-c88bf0d99c4d)
Same issue here

**Reply by CberYellowstone** [reply]
Same here

**Reply by liuliangsir** [reply]
![education github com_discount_requests_application](https://github.com/community/community/assets/8049878/b260c648-d61e-4a88-a08c-459bd0719de1)

**Comment by Respawn-ID** [comment]
I wish you all a Happy New Year too 🎆



---Original---
From: ***@***.***&gt;
Date: Mon, Jan 1, 2024 15:45 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Congrats! Your request for @yourname wasapproved and your benefits will be available soon (Discussion #58094)




 
And I wish you all a Happy New Year 🎆

**Comment by Respawn-ID** [comment]
yep,you need to wait for few day's and normally it takes 4days or even more



---Original---
From: ***@***.***&gt;
Date: Tue, Jan 2, 2024 22:31 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Congrats! Your request for @yourname wasapproved and your benefits will be available soon (Discussion #58094)




 
have you received the benefit?

**Comment by Respawn-ID** [comment]
good luck and wish you get your benefits soon



---Original---
From: "Zekun ***@***.***&gt;
Date: Tue, Jan 16, 2024 09:45 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Congrats! Your request for @yourname wasapproved and your benefits will be available soon (Discussion #58094)




 
I hope today i will access my benefits, Im waiting for 4days……

**Comment by Respawn-ID** [comment]
yep l tried almost like you



---Original---
From: "Yang ***@***.***&gt;
Date: Mon, Mar 4, 2024 19:49 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Congrats! Your request for @yourname wasapproved and your benefits will be available soon (Discussion #58094)




 
I tried 29 times before passing

**Comment by ddy7918** [comment]
waiting for mine

**Comment by celikcedev** [comment]
I'm also waiting for the benefits to be assigned.It approved at 09.06.2024

**Comment by samus-aran** [comment, marked as answer]
Hi everyone 👀 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

We will be locking this discussion. If you require guidance on your application please open a new discussion (preferably after you've read the FAQ 😉 )
